tests/integration/pox/stacking/direct-stacking-good-order.spec.ts

```diff
@@ -98,25 +91,31 @@ describe("testing solo stacker increase without bug", () => {
 
     // Asserts about pox info for better knowledge sharing
     expect(poxInfo.contract_id).toBe("ST000000000000000000002AMW42H.pox-2");
-    expect(poxInfo.pox_activation_threshold_ustx).toBe(50_286_942_145_278);
+    // expect(poxInfo.pox_activation_threshold_ustx).toBe(50_286_942_145_278);
     expect(poxInfo.current_cycle.id).toBe(1);
-    expect(poxInfo.current_cycle.min_threshold_ustx).toBe(20_960_000_000_000);
+    // expect(poxInfo.current_cycle.min_threshold_ustx).toBe(20_960_000_000_000);
 
     // Assert that the next cycle has 100m STX locked
     expect(poxInfo.current_cycle.stacked_ustx).toBe(0);
     expect(poxInfo.current_cycle.is_pox_active).toBe(false);
     expect(poxInfo.next_cycle.stacked_ustx).toBe(100_000_000_000_111);
 
-    const poxAddrInfo0 = await readRewardCyclePoxAddressList(network, 2, 0);
-    const poxAddrInfo0CV = hexToCV(poxAddrInfo0.data) as SomeCV<TupleCV>;
-    const poxAddrInfoStr0 = cvToString(hexToCV(poxAddrInfo0.data));
+    const poxAddrInfo0 = (await readRewardCyclePoxAddressForAddress(
+      network,
+      2,
+      Accounts.WALLET_2.stxAddress
+    )) as Record<string, ClarityValue>;
```

Comment:
It would be nice to have a real type here.

Reply:
Agreed.